tests/test_jupyter_tutorial.py

```diff
@@ -0,0 +1,72 @@
+from cookiecutter.main import cookiecutter
+import os
+import filecmp
+import shutil
+import filecmp
+
+
+
+def download_tutorial():
+    try:
+        cookiecutter(
+            "https://github.com/SebChw/art_template.git",
+            no_input=True,
+            extra_context={"project_name": "mnist_tutorial", "author": "test", "email": "test"},  # Pass the project_name to the template,
+            checkout="mnist_tutorial_cookiecutter",  # Use the latest version of the template
```

Comment:
IDK if this is the way we want it. If so, we'd need to first merge a new version of the tutorial and then merge this one. Maybe the script should look into the comments look for the art_template link and then take the branch from the link.

Reply:
Not sure about it as well. However we should remember that this is a test of ART not the test of a notebook. At least this is what I was aiming to achieve.